frontend/public/components/events.jsx

```diff
@@ -280,8 +280,8 @@ class EventStream extends SafetyFirst {
         this.setState({error: false, loading: false, sortedMessages: [], filteredEvents: []});
       })
       .onclose(evt => {
-        if (evt && evt.wasClean === false) {
-          this.setState({error: evt.reason || 'Connection did not close cleanly.'});
+        if (evt) {
+          this.setState({error: evt.reason || 'The terminal connection has closed.'});
```

Comment:
Hold on, this is the events stream, right? Not terminal? It looks like this is the wrong place for this change.

Reply:
Ok, reverted `events.jsx`